ragna-docker.toml

```diff
@@ -0,0 +1,22 @@
+local_cache_root = "/var/ragna"
+document = "ragna.core.LocalDocument"
+authentication = "ragna.deploy.RagnaDemoAuthentication"
+
+[components]
+source_storages = [
+    "ragna.source_storages.Chroma",
+    "ragna.source_storages.RagnaDemoSourceStorage",
+    "ragna.source_storages.LanceDB"
+]
+assistants = [
+    "ragna.assistants.RagnaDemoAssistant"
+]
```

Comment:
These are all the builtin components that we can activate by default. For everything else we need some input from the user, i.e. environment variables. It sucks, that we can't include everything here and "deactivate" the components for which the requirements are not met. Will open an issue about this.